.github/workflows/release-lib-injection.yml

```diff
@@ -1,33 +1,32 @@
+# name: "Release Library Injection"
```

Comment:
I suggest leaving a comment in code to remind us to re-enable this, and when we should re-enable it

Reply:
I forgot to push